problem_set_1.py

```diff
@@ -62,7 +89,17 @@
 # eo - Saluton, <name>!
 #
 # If any other language code is used, return nothing.
-
+def multigreeting(name, language):
+    if language == "en":
```

Comment:
This is awesome. You could consider refactoring it with a dictionary mapping languages to output.